src/main/java/ui/BoardAutoCreator.java

```diff
@@ -101,6 +127,26 @@ private void createWorkAllocationBoard() {
                         "It is saved under the name \"" + boardName + "\".");
     }
 
+    private void createSampleBoard() {
+        logger.info("Creating " + SAMPLE_BOARD);
+
+        panelControl.closeAllPanels();
+
+        List<PanelInfo> panelData = new ArrayList<>();
+
+        panelData.add(new PanelInfo(SAMPLE_PANEL_NAMES.get(0), SAMPLE_PANEL_FILTERS.get(0)));
+        panelData.add(new PanelInfo(SAMPLE_PANEL_NAMES.get(1), SAMPLE_PANEL_FILTERS.get(1)));
+        panelData.add(new PanelInfo(SAMPLE_PANEL_NAMES.get(2), SAMPLE_PANEL_FILTERS.get(2)));
```

Comment:
Use a loop for these? For tests too. That was the reason for changing to an array.